config/samples/cluster_with_backup.yaml

```diff
@@ -21,6 +21,28 @@
 # If you are testing this in Docker Desktop, you can browse the local MinIO
 # instance at https://localhost:9000. Note: This will use a certificate that
 # is not in your browser's trust store, so you will get a security warning.
+#
+# If you want to test a restore, you can take the following steps:
```

Comment:
I wonder if the user manual (or user manual backup) would be a better place to document these steps?

Reply:
Eventually it will, and there's a separate issue on that: #174